templates/shared-v1/contextual_footers/_desktop_download.html

```diff
@@ -1,5 +1,7 @@
 <div class="col-4 p-divider__block">
-  <img src="{{ ASSET_SERVER_URL }}be3876ec-picto-download-warmgrey.svg" alt="Download pictogram" />
+  <div class="u-align--center u-padding-bottom">
+    <img src="{{ ASSET_SERVER_URL }}be3876ec-picto-download-warmgrey.svg" alt="Download pictogram" style="height: 90px;" />
```

Comment:
This inline CSS is repeated a lot. Could it be a class? We should only use inline styles where absolutely necessary otherwise we'll have a difficult time maintaining the codebase.